django_guid/config.py

```diff
@@ -36,6 +34,15 @@ def __init__(self) -> None:
                 raise ImproperlyConfigured('RETURN_HEADER must be a boolean')
             if not isinstance(self.EXPOSE_HEADER, bool):
                 raise ImproperlyConfigured('EXPOSE_HEADER must be a boolean')
+
+            # SKIP_CLEANUP was deprecated in v1.2.0 and should be removed completely in v2.0.0
+            if 'SKIP_CLEANUP' in _settings:
+                # flake8: noqa: T001
+                print(
```

Comment:
Haven't really looked into the PR, but deprecation warnings are done like this: 
```suggestion
                import warnings
                warnings.warn('string', DeprecationWarning)
```

Reply:
From what I can tell `PendingDeprecationWarning` would be most appropriate, so I've used that.

Reply:
I’m reading the docs differently. `PendingDeprecationWarning` is used when you know in advance something is going to be deprecated in a later version, but is not deprecated at this time.

Reply:
Yep, changed to DeprecationWarning